v2/dtos/common/base.go

```diff
@@ -28,3 +28,11 @@ type BaseResponse struct {
 type Versionable struct {
 	ApiVersion string `json:"apiVersion,omitempty"`
 }
+
+// BaseWithIdResponse defines the base content for response DTOs (data transfer objects).
+// This object and its properties correspond to the BaseWithIdResponse object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/BaseWithIdResponse
+type BaseWithIdResponse struct {
```

Comment:
Once you rebase, please add factory method for this similar to the new ones I added. THX.

Reply:
Rebased branch and added factory methods to the response DTOs.